nvim/install.sh

```diff
@@ -25,9 +25,8 @@ tar -C ~/.nvim --extract -z -f "${tmpfile}" --strip-components 1
 script_dir=$(dirname "$0")
 . "${script_dir}/path.zsh"
 
-source "$DOTS/common/pip.sh"
-ensure_pip_installed
-sudo python3 -m pip install --upgrade pynvim
+source "$DOTS/common/python.sh"
```

Comment:
Fails in CI but does not fail CI

Reply:
https://github.com/kaihowl/dotfiles/runs/8264063776?check_suite_focus=true#step:4:1389

This seems to have since been fixed upstream. If there is no wheel, we fallback.

Reply:
I also fixed this by explicitly installing wheel now.